backend/oasst_backend/user_stats_repository.py

```diff
@@ -227,6 +227,9 @@ def get_trollboard(
         if enabled is not None:
             qry = qry.filter(User.enabled == enabled)
 
+        if lang is not None and not "":
+            qry = qry.filter(User.most_used_lang(session=self.session) == lang)
+
         qry = qry.order_by(TrollStats.rank).limit(limit)
```

Comment:
The querry : 
`SELECT "user".id AS user_id, "user".username AS user_username, "user".auth_method AS user_auth_method, "user".display_name AS user_display_name, "user".last_activity_date AS user_last_activity_date, "user".enabled AS user_enabled, "user".deleted AS user_deleted, "user".show_on_leaderboard AS user_show_on_leaderboard, troll_stats.user_id AS troll_stats_user_id, troll_stats.base_date AS troll_stats_base_date, troll_stats.modified_date AS troll_stats_modified_date, troll_stats.time_frame AS troll_stats_time_frame, troll_stats.troll_score AS troll_stats_troll_score, troll_stats.rank AS troll_stats_rank, troll_stats.red_flags AS troll_stats_red_flags, troll_stats.upvotes AS troll_stats_upvotes, troll_stats.downvotes AS troll_stats_downvotes, troll_stats.spam_prompts AS troll_stats_spam_prompts, troll_stats.quality AS troll_stats_quality, troll_stats.humor AS troll_stats_humor, troll_stats.toxicity AS troll_stats_toxicity, troll_stats.violence AS troll_stats_violence, troll_stats.helpfulness AS troll_stats_helpfulness, troll_stats.spam AS troll_stats_spam, troll_stats.lang_mismach AS troll_stats_lang_mismach, troll_stats.not_appropriate AS troll_stats_not_appropriate, troll_stats.pii AS troll_stats_pii, troll_stats.hate_speech AS troll_stats_hate_speech, troll_stats.sexual_content AS troll_stats_sexual_content, troll_stats.political_content AS troll_stats_political_content
FROM "user" JOIN troll_stats ON "user".id = troll_stats.user_id
WHERE troll_stats.time_frame = %(time_frame_1)s ORDER BY troll_stats.rank
 LIMIT %(param_1)s`